test/unit_test/components/arm_test.dart

```diff
@@ -0,0 +1,290 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:grpc/grpc.dart';
+import 'package:viam_sdk/src/components/arm/service.dart';
+import 'package:viam_sdk/src/gen/component/arm/v1/arm.pbgrpc.dart';
+import 'package:viam_sdk/src/resource/manager.dart';
+import 'package:viam_sdk/src/utils.dart';
+import 'package:viam_sdk/viam_sdk.dart';
+
+import '../../test_utils.dart';
+
+class FakeArm extends Arm {
+  bool isStopped = true;
+  JointPositions armJoinPositions = JointPositions(values: [0, 0, 0]);
```

Comment:
small thing, but I was curious why it's named `JoinPositions`?

Reply:
that is a typo, thank you for finding that :)